extendr-api/src/wrapper/dataframe.rs

```diff
@@ -0,0 +1,64 @@
+use super::*;
+
+pub trait IntoDataFrame<T> {
```

Comment:
Not sure if I've asked this yet, but why not just `TryInto<DataFrame>`? Why create a new trait?

Reply:
I think it should be fine? It would be implementing a foreign trait `TryInto` on a local type, it's just that the local type is generic.